.goreleaser.yml

```diff
@@ -9,6 +9,8 @@ builds:
      - arm64
    env:
      - CGO_ENABLED=0
+   ldflags:
+     - -s -w -X github.com/tufin/oasdiff/build.Version={{.Version}}
```

Comment:
Isn't it sufficient to use:
- -s -w -X build.Version={{.Version}}

This way we decouple from the repo location and name.

Reply:
not when using go modules, it depends on this line https://github.com/Tufin/oasdiff/blob/a39d70503d88953d2f4c0c2886b3cb9ffce0e263/go.mod#L1

This is also why it works for my fork